### PR TITLE
fix: roles section overlapping with ticket section

### LINKED ISSUE #6057: Roles section overlaps with the Ticket section
**Describe the bug**
In the users event section there are few columns named as name, date, roles, tickets and etc. If the email addresses mentioned under the roles section are long they will overlap with the tickets section. 

**To Reproduce**
1. Click on 'Combo box near the singed in users email address'
2. Click on 'Manage Events'
3. See error

**Expected behavior**
Email address should not overlap with the tickets section. Should use proper overflow style.

**Screenshots**

image

**Desktop:**
 - OS: Windows 10
 - Browser: Chrome
 - Version :  87.0.4280.88 (Official Build) (64-bit)

### PR DESCRIPTION
**Pull request by Creatoon**
Fixes #6057 

#### Short description of what this resolves:
The roles section's email was overlapping with ticket section , This PR resolves this bug, screenshot is available below.

BEFORE

bef

AFTER

Screenshot_2020-12-18 Live Events eventyay(1)

#### Checklist

- [ yes] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [ yes] My branch is up-to-date with the Upstream `development` branch.
- [ Not Checked] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ yes] I have added tests that prove my fix is effective or that my feature works
- [ Not required] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by Creatoon** [comment]
@mariobehling @iamareebjamal Can you review this PR.

File: app/templates/components/ui-table/cell/cell-roles.hbs [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
   {{/each}}
   {{#each this.extraRecords.registrars as |role|}}
-    <div class="item">{{role.email}} ({{t 'Registrars'}})</div>
+  <div class="item word-break">{{role.email}} ({{t 'Registrars'}})</div>
   {{/each}}
   {{#each this.extraRecords.moderators as |role|}}
-    <div class="item">{{role.email}} ({{t 'Moderators'}})</div>
+  <div class="item word-break">{{role.email}} ({{t 'Moderators'}})</div>
   {{/each}}
 </div>
```
Comment (daretobedifferent18) [inline app/templates/components/ui-table/cell/cell-roles.hbs:18]: add line at end of file

**Comment by daretobedifferent18** [comment]
@iamareebjamal It is working fine for me

File: app/templates/components/ui-table/cell/cell-roles.hbs [hunk 2]
```diff
@@ -15,4 +15,4 @@
   {{#each this.extraRecords.moderators as |role|}}
   <div class="item word-break">{{role.email}} ({{t 'Moderators'}})</div>
```
Comment (iamareebjamal) [inline app/templates/components/ui-table/cell/cell-roles.hbs:16]: indentation

File: app/templates/components/ui-table/cell/cell-roles.hbs [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
-  {{#each this.extraRecords.registrars as |role|}}
-    <div class="item">{{role.email}} ({{t 'Registrars'}})</div>
-  {{/each}}
-  {{#each this.extraRecords.moderators as |role|}}
-    <div class="item">{{role.email}} ({{t 'Moderators'}})</div>
-  {{/each}}
-</div>
\ No newline at end of file
+  <div class="item word-break">{{this.record.email}} ({{t 'Owner'}})</div>
+    {{#each this.extraRecords.organizers as |role|}}
```
Comment (daretobedifferent18) [inline app/templates/components/ui-table/cell/cell-roles.hbs:3]: still indentation is wrong. Pls fix this

Comment (Creatoon) [reply app/templates/components/ui-table/cell/cell-roles.hbs:3]: @daretobedifferent18 please tell me what is the correct indentation for this file , don't i have to indent "#each" statements

Comment (daretobedifferent18) [reply app/templates/components/ui-table/cell/cell-roles.hbs:3]: just add word-break class and change nothing.

**Review by daretobedifferent18 — APPROVED** [review]
code looks clean now

**Pull request merged**